hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -4429,4 +4429,13 @@
       maximum number of buckets across all volumes.
     </description>
   </property>
+  <property>
+    <name>ozone.om.requests.ignore.terminate.on.failure</name>
```

Comment:
Should we provide this as a config or use it as a constant in code itself. Misuse of this property could be dangerous and lead to consistent state. By providing a config,  the user/admin has control over internal behaviour of OM. IMO we should not provide such a config as providing such options will be more prone to problems.